js/page-specific/citysourced-viewer.js

```diff
@@ -1,153 +1,160 @@
 
-baltimoreCounty.pageSpecific.citySourcedViewer = (function($, querystringer, maps, undefined) {
+baltimoreCounty.pageSpecific.citySourcedViewer = (function viewer($, querystringer, moment) {
 	'use strict';
 
-	var getNearbyData = function(settings, callback) {
-		
-
-			$.ajax('//testservices.baltimorecountymd.gov/api/citysourced/getreportsbylatlng', {
-				method: 'POST',
-				data: settings
+	var getNearbyData = function getNearbyData(settings, callback) {
+		$.ajax('//testservices.baltimorecountymd.gov/api/citysourced/getreportsbylatlng', {
+			method: 'POST',
+			data: settings
+		})
+			.done(function done(data) {
+				if (callback) {
+					callback(data);
+				}
 			})
-				.done(function(data) {
-					if (callback)
-						callback(data);
-				})
-				.fail(function(error) {
+			.fail(function fail(error) {
+				console.log(error);
+			});
+	};
 
-				});
-		},
-
-		/**
-		 * Descending date sort.
-		 */
-		commentDateComparer = function(a, b) {
-			var aMoment = moment(a.Created),
-				bMoment = moment(b.Created);
-
-			if (aMoment.isAfter(bMoment)) 
-				return -1;
-			else 
-				return 1;
-		},
-
-		/**
-		 * Hides all but the top `numberToShow` comments in the `$target` list.
-		 */
-		hideMostComments = function($target, commentData, numberToShow) {
-			if (commentData && commentData.length > numberToShow) {
-				$target.find('li').not(function(index, element) {
-					return index <= numberToShow - 1;
-				}).hide();
-				$target.after('<p><a href="javascript:;" id="show-comments" aria-role="button">Load ' + (commentData.length-numberToShow) +' more comments</a></p>');
-				$('#show-comments').on('click', function() {
-					$target.find('li').slideDown(500);
-					$(this).hide();
-				});	
-			}
-		},
-
-		/**
-		 * Removes seconds from the date strings of a data item.
-		 */
-		processTimeForViewerDataItem = function(data) {
-
-			if (data.DateCreated)
-				data.DateCreated = removeSeconds(data.DateCreated);
-			
-			if (data.DateUpdated)
-				data.DateUpdated = removeSeconds(data.DateUpdated);
-
-			if (data.Comments) {
-				for (var n = 0; n < data.Comments.length; n++) {
-					data.Comments[n].Created = removeSeconds(data.Comments[n].Created);
-				}
-			}
+	/**
+	 * Descending date sort.
+	 */
+	var commentDateComparer = function commentDateComparer(a, b) {
+		var aMoment = moment(a.Created);
+		var bMoment = moment(b.Created);
+
+		if (aMoment.isAfter(bMoment)) {
+			return -1;
+		}
+		return 1;
+	};
+
+	/**
+	 * Hides all but the top `numberToShow` comments in the `$target` list.
+	 */
+	var hideMostComments = function hideMostComments($target, commentData, numberToShow) {
+		if (commentData && commentData.length > numberToShow) {
+			$target.find('li').not(function lisItemHider(index) {
+				return index <= numberToShow - 1;
+			}).hide();
+			$target.after('<p><a href="javascript:;" id="show-comments" aria-role="button">Load ' + (commentData.length - numberToShow) + ' more comments</a></p>');
```

Comment:
Not as such, no, but I'm open to suggestions. If this was ES6 using Babel, I'd use interpolation, but that's not really an option here.